color/src/lib.rs

```diff
@@ -9,13 +9,49 @@
 // shouldn't apply to examples and tests
 #![warn(unused_crate_dependencies)]
 #![warn(clippy::print_stdout, clippy::print_stderr)]
-// TODO: parts of the crate are not done, with some missing docstring,
-// and some enum variants not yet implemented. Finish those and remove
-// these allow attributes.
-#![allow(clippy::todo, reason = "need to fix todos")]
 
 //! Color is a Rust crate which implements color space conversions, targeting at least CSS4 color.
 //!
+//! ## Scope and goals
+//!
+//! Color in its entirety is an extremely deep and complex topic. It is completely impractical
+//! for a single crate to meet all color needs. The goal of this one is to strike a balance,
+//! providing color capabilities while also keeping things simple and efficient.
+//!
+//! The main purpose of this crate is to provide a good set of types for representing colors,
+//! along with conversions between them and basic manipulations, especially interpolation. A
+//! major inspiration is the CSS Color Level 4 draft spec; we implement most of the operations
+//! and strive for correctness.
+//!
+//! Simplifications include:
+//!   * Always using `f32` to represent component values.
+//!   * Only handling 3-component color spaces (plus optional alpha).
+//!   * Choosing a fixed, curated set of color spaces for dynamic color types.
+//!   * Choosing linear sRGB as the central color space.
+//!   * Keeping white point implicit.
+//!
+//! A number of other tasks are out of scope for this crate:
+//!   * Print color spaces (CMYK).
+//!   * Spectral colors.
+//!   * Color spaces with more than 3 components generally.
+//!   * [ICC] color profiles.
+//!   * [ACES] color transforms.
+//!   * Appearance models and other color science not needed for rendering.
```

Comment:
I think this is a slightly buried lede; we should be upfront that our goal is computer graphics rendering.